conda_package/mpas_tools/scrip/from_mpas.py

```diff
@@ -47,10 +44,13 @@ def scrip_from_mpas(mpasFile, scripFile, useLandIceMask=False):
     sphereRadius = float(fin.sphere_radius)
     on_a_sphere = str(fin.on_a_sphere)
 
+    # change the longitude convention to use positive values [0 2pi]
+    lonCell[lonCell<0.0] += 2.0*np.pi
+    lonVertex[lonVertex<0.0] += 2.0*np.pi
+
     if sphereRadius <= 0:
-        print(" -- WARNING: conservative remapping is NOT possible when "
-              "'sphereRadius' <= 0 because 'grid_area' field will be infinite "
-              "(from division by 0)")
+       sphereRadius =  6.37122e6
```

Comment:
I would prefer to get this from `mpas_tools.cime.constants` rather than hard coding it:
https://github.com/MPAS-Dev/MPAS-Tools/blob/master/conda_package/mpas_tools/cime/constants.py#L6

We will need to add a new import at the top:
```
from mpas_tools.cime.constants import constants
```
and then make this change:
```suggestion
       sphereRadius =  constants['SHR_CONST_REARTH']
```